cetav-ddw/lalibertad-cetav
# pagina de contacto

## Description

In this pull request, I have created a rough draft of how the contact page should look. This includes the design of information boxes for various contact details and an example of how the map image should be positioned.

### How to verify this PR

To verify these changes, follow these steps:

1. Check the design of the contact information boxes for each contact method. They should be displayed in a column layout on the left side of the page.

2. Ensure that each information box contains the appropriate contact details, such as phone number, email, Facebook, Instagram, and address.

3. Verify that the map image is displayed on the right side of the page and is centered within its container.

![ejemplo](https://github.com/cetav-ddw/lalibertad-cetav/assets/84554799/1ac0b5e4-1ef2-4918-8f46-8eea5efee342)
